Formula/monit.rb

```diff
@@ -20,10 +20,36 @@ def install
                           "--localstatedir=#{var}/monit",
                           "--sysconfdir=#{etc}/monit",
                           "--with-ssl-dir=#{Formula["openssl@1.1"].opt_prefix}"
+    system "make"
```

Comment:
Doesn't it build without this?

Reply:
Actually, it does, but `make` command is listed in the official wiki so I suppose it just happens to work without it.
Maybe it is better to add it. Any opinions?

Reply:
Just let me know if you think it is better to remove it or not.

Reply:
Leave it.

Reply:
OK, I have squashed the commits. Just ping me if there is anything else I need to do.

Thanks for creating and maintaining homebrew. :-)